packages/ckeditor5-image/tests/imagestyle/imagestyleediting.js

```diff
@@ -176,7 +176,7 @@ describe( 'ImageStyleEditing', () => {
 				plugins: [ ImageBlockEditing, ImageStyleEditing ]
 			} );
 
-			sinon.assert.calledOnceWithMatch( normalizationSpy, {
+			expect( normalizationSpy.firstCall.args[ 0 ] ).to.deep.equal( {
```

Comment:
This change is related to https://github.com/toss/es-toolkit/issues/953 because `sinon` internally tries to use `toString` which is not available. IMO, current `es-toolkit` behavior makes more sense than in `lodash-es`.